common/src/main/java/org/mcaccess/minecraftaccess/MainClass.java

```diff
@@ -55,21 +57,12 @@ public static void init() {
         if (MainClass.getScreenReader() != null && MainClass.getScreenReader().isInitialized())
             MainClass.getScreenReader().say(msg, true);
 
-        MainClass.inventoryControls = new InventoryControls();
-        MainClass.biomeIndicator = new BiomeIndicator();
-        MainClass.xpIndicator = new XPIndicator();
-        MainClass.facingDirection = new FacingDirection();
-        MainClass.playerStatus = new PlayerStatus();
-        MainClass.playerWarnings = new PlayerWarnings();
-        MainClass.accessMenu = new AccessMenu();
-        MainClass.fluidDetector = new FluidDetector();
-        speakHeldItem = new SpeakHeldItem();
-
         for (KeyMapping km : KeyBindingsHandler.getInstance().getKeys()) {
             KeyMappingRegistry.register(km);
         }
 
         ClientTickEvent.CLIENT_POST.register(MainClass::clientTickEventsMethod);
+        ClientPlayerEvent.CLIENT_PLAYER_JOIN.register(player -> initWorldState(player));
```

Comment:
A method refrence can be used here.
```suggestion
        ClientPlayerEvent.CLIENT_PLAYER_JOIN.register(MainClass::initWorldState);
```